sass/main.scss

```diff
@@ -940,6 +938,57 @@ footer {
         grid-template-columns: repeat(4, $leadership-size-mobile);
         justify-content: space-between;
     }
+
+}
+
+@media screen and (min-width: 1700px) {
```

Comment:
Please fix all indentation: 4 space throughout!

Reply:
Also could you move this entire `@media` query to be the first media query? That way it goes largest screen -> smallest screen target.

Reply:
[media query movement](https://github.com/codeRIT/brickhack.io/pull/1053/commits/0c7065e5da04c118920cccd5cbda026c2aa7c512)

spacing should be fixed on the most recent commit